examples/discovery-search-app/cypress/integration/collection_filter/spec.ts

```diff
@@ -0,0 +1,142 @@
+describe('Collection Filter', () => {
+  beforeEach(() => {
+    //TODO: swap this section with the helper function, once that code is merged in
+    // Sets up and handles the collections, component settings, and initial query requests that run on page-load
+    cy.server();
+    cy.fixture('collections/collections.json').as('collectionsJSON');
+    cy.route('GET', '**/collections?version=2019-01-01', '@collectionsJSON').as('getCollections');
+    cy.fixture('component_settings/componentSettings.json').as('componentSettingsJSON');
+    cy.route('GET', '**/component_settings?version=2019-01-01', '@componentSettingsJSON').as(
+      'getComponentSettings'
+    );
+    cy.fixture('query/query.json').as('queryJSON');
+    cy.route('POST', '**/query?version=2019-01-01', '@queryJSON').as('postQuery');
+    cy.visit('/');
+    cy.wait(['@getCollections', '@getComponentSettings', '@postQuery']);
+
+    // Set up/override routes & fixtures that are specific to this file
+    cy.fixture('query/singleCollectionQuery.json').as('singleCollectionQueryJSON');
+    cy.fixture('query/doubleCollectionQuery.json').as('doubleCollectionQueryJSON');
+  });
+
+  describe('when the example app loads', () => {
+    it('the collection facet select does not appear', () => {
+      cy.get('.collection-facet-select').should('not.exist');
```

Comment:
i would be careful about doing a `not.exist` check prior to at least one `exist` call - this would pass probably immediately without something else to look for on the page

Reply:
That's a good call. I'll add something to check for.

Reply:
Actually, looking closer at this test, it isn't even working. `collection-facet-select` is the id of the component, not a classname. And the collection facet selector *is* visible on page load; this test was just written before I had the `wait`s for initial requests in place.

I think I'll switch this test to "the collection facet selector ~does not~ appears."